### DISCUSSION TITLE
Welcome to ZuluSCSI-Pico-OSHW Discussions!

### DISCUSSION THREAD

**Discussion by aperezbios** [ZuluSCSI/ZuluSCSI-Pico-OSHW#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below with an introduction of yourself and tell us about what interests you about ZuluSCSI Pico

**Comment by mdevaev** [comment]
Congratulations with launch!

**Comment by xenocide21** [comment]
Already waiting for it to arrive in the mail!
